selenium_tests/account_admin/page_objects/account_admin_dashboard_page_object.py

```diff
@@ -10,8 +10,7 @@ class AccountAdminDashboardPageLocators(object):
     # use text() for exact match (should only match on dashboard page)
     PAGE_TITLE = (By.XPATH, '//h3[text()="Admin Console"]')
     COURSE_INFO_LINK = (By.XPATH, "//a[contains(.,'Find Course Info')]")
-    CROSS_LISTING_DIV = (By.ID, "cross-listing-button")
-
+    CROSS_LISTING_DIV = (By.XPATH, "//a[contains(., 'Cross-list Courses')]")
```

Comment:
Because that element is a really a link. I totally get what you're saying and had initially set it up initially to look it up by id as well but that had returned an element not found a handful of times (unexpectedly).  I haven't run into the same issue using the anchor tag instead, so I decided to opt for it instead for this case in favor of dealing with flakiness.  Can always change it, if need be.